q2_amr/card/tests/test_utils.py

```diff
@@ -20,9 +19,13 @@
 class TestAnnotateReadsCARD(TestPluginBase):
     package = "q2_amr.card.tests"
 
-    @classmethod
-    def setUpClass(cls):
-        cls.allele_count_df = pd.DataFrame(
+    def setUp(self):
```

Comment:
Why did you change this to not be a class method anymore? Now all those variables will need to be re-initialized before every test. If if was just for the sake of doing `self.tmp = self.temp_dir.name` you could instead just do this inside your single test which is using the temp_dir.

Reply:
Yes that makes sense 👍